src/Web/wwwroot/app/accounts/views/accountsLoginInfo.html

```diff
@@ -8,7 +8,7 @@
     </div>
     <div class="form-group has-feedback" show-errors>
         <label for="email" class="sr-only">Email</label>
-        <input type="email" id="email" name="Email" class="form-control" placeholder="Email" ng-model="model.email"
+        <input type="email" id="email" name="Email" class="form-control" placeholder="Email" ng-model="model.email" autofocus
```

Comment:
Should work. Most likely requires this as well on the controller.
https://gist.github.com/mlynch/dd407b93ed288d499778#file-autofocus-js

Either that or this as a global. 
https://gist.github.com/kirkstrobeck/599664399dbc23968741